spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :email }
+    it { is_expected.to have_db_column :encrypted_password }
+  end
+
+  describe 'FactoryBot' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:user)).to be_valid
```

Comment:
Please include FactoryBot Matchers in your RSpec configuration (And Cucumber configuration) so that you can shorten this to
```
expect(create(:article)).to be_valid 
```